bin/jobsub_submit

```diff
@@ -193,10 +193,11 @@ def main():
     #    shutil.copyfile(token, token_dest)
     #    varg["token"] = token_dest
 
+    varg["is_dag"] = True
```

Comment:
I understand why you're setting `varg["is_dag"]` this way, but I worry that six months from now, we'll see this line and wonder why we've "defaulted" to setting `varg["is_dag"]` to `True`.  Would it be possible, if slightly less elegant to do something along the lines of:

```python
varg["is_dag"] = False
if args.dag or args.dataset_definition or args.maxConcurrent:
    varg["is_dag"] = True
```
or even 

```python
varg["is_dag"] = True if (args.dag or args.dataset_definition or args.maxConcurrent) else False
```

And then having the `if` block on 197?

I could even see having line 196 be 
```python
varg["is_dag"] = False
```
and then setting `varg["is_dag"]` to `True` for every applicable clause in the `if` statement starting in line 197.  

I know this is extra code compared to what you wrote here, I just worry about the future understandability of this bit.

Reply:
Yeah - it's certainly _nicer_ the way you did it here, but I do have concerns about the clarity.